examples/core/cubemap/app.js

```diff
@@ -1,14 +1,20 @@
 /* global document */
-import {AnimationLoop, GL, TextureCube, Cube, Matrix4, radians} from 'luma.gl';
+import {
+  AnimationLoop, GL, TextureCube, Cube, Matrix4, radians,
+  resetParameters, setParameters
+} from 'luma.gl';
 
 const animationLoop = new AnimationLoop({
   onInitialize: ({gl, canvas}) => {
     addControls(canvas);
 
-    gl.clearColor(0, 0, 0, 1);
-    gl.clearDepth(1);
-    gl.enable(GL.DEPTH_TEST);
-    gl.depthFunc(GL.LEQUAL);
+    resetParameters(gl);
```

Comment:
My opinion is `reset` should be `true` by default, some one writing new example would like to start with default state, moreover one should not depend on existing state given user might click on examples in any order.